specs/subresourceintegrity/spec.markdown

```diff
@@ -310,18 +297,17 @@ resource in order to provide agility in the face of future discoveries.
 For example, the "Hello, world." resource described above may be described
 either of the following `ni` URLs:
 
-    ni:///sha-256;-MO_YqmqPm_BYZwlDkir51GTc9Pt9BvmLrXcRRma8u8?ct=text/plain
-    ni:///sha-512;rQw3wx1psxXzqB8TyM3nAQlK2RcluhsNwxmcqXE2YbgoDW735o8TPmIR4uWpoxUERddvFwjgRSGw7gNPCwuvJg?ct=text/plain
+    ni:///sha-256;-MO_YqmqPm_BYZwlDkir51GTc9Pt9BvmLrXcRRma8u8?ct=application/javascript
+    ni:///sha-512;rQw3wx1psxXzqB8TyM3nAQlK2RcluhsNwxmcqXE2YbgoDW735o8TPmIR4uWpoxUERddvFwjgRSGw7gNPCwuvJg?ct=application/javascript
 {:.example.highlight}
```

Comment:
Good point. I've created #144 for this since it's a bigger change than just removing support for the anchor element. Feel free to assign it to me.